### DISCUSSION TITLE
18650 Battery Addon Help

### DISCUSSION THREAD

**Discussion by majorpbx** [sanni/cartreader#136] [Need help]
First of all thank you for creating a wonderful board! I recently bought the 18650 Battery Addon and what i got looks to be a much newer version. I'll attach pictures of it to help but I'm unable to locate "S1" (+ to Arduino) from step 2 from [here](https://github.com/sanni/cartreader/wiki/Adding-a-battery). I wanted to confirm the voltage and current that should be expected from this connection point. If not help identify a new connection point for my particular board.

![Bottom](https://user-images.githubusercontent.com/18235905/116421918-44f6d380-a80d-11eb-982a-ffcd8eb123fd.jpg)
![Top](https://user-images.githubusercontent.com/18235905/116421957-4b854b00-a80d-11eb-8861-b26f2b9793a5.jpg)

Thanks

**Comment by sanni** [comment]
It's the 5V pin of the micro USB port. But since the pins on the USB port are so tiny I followed the trace to the next big solder pad which in my case was the S1 fuse.

**Reply by majorpbx** [reply]
Gotcha, so then in that top picture I need to use one of the 5V -/+ connection spots for the Cart Reader then correct?

Just curious, why wouldn't you use two 5V rails for the Arduino and the Cart Reader?

**Comment by sanni** [comment]
Correct. 

The voltage goes from the Arduino's USB port, which has been disconnected from the Arduino by removing the two parts from the Arduino PCB, to the input of the battery PCB to charge the battery.

Then from the output of the battery PCB it goes to the Cart Reader PCB and the Cart Reader PCB finally supplies the Arduino over the pin headers.

The USB ports on the battery PCB are not used.

**Reply by majorpbx** [reply]
Got it, makes sense and thank you for your replies.

**Comment by majorpbx** [comment]
Just in case anyone else gets this board. The white switch in the top right part of the bottom picture will not allow it to fit comfortably inside the full case. This switch is important as this board automatically shuts off after 10 minutes of inactivity. If you de-solder it (which I recommend for correct case fit) all you would need is a tactile switch (default open) soldered with wire and attached to the case somewhere to control the battery module. This switch can turn the battery module on and off by connecting the positive lead to ground briefly.

1 push = on
double tap = off

**Comment by majorpbx** [comment]
Here is the layout for anyone interested that gets this model. Something else I ran into is that the 2R2 component is a bit higher than the top of the full case allows. ;)
![Bottom_with_connections](https://user-images.githubusercontent.com/18235905/122089183-826eea80-cdd4-11eb-9b0d-d97f53db5c60.jpg)

**Comment by majorpbx** [comment]
*update*

I've modified the case top with battery addition to hopefully fit these battery boards if anyone is interested. Also expanded the SNES connector slot a bit to help it fit better.

Made more changes to the top if anyone is interested. Added battery charge window and another hole for a top mounted button on the battery board. You can use the same as the Sanni Cart Board tactile switches in the "Switch Lead" locations noted in the pictures. When soldering clip off the lead that would go to "- Switch Lead" and it will work great! Also, you can add another button to print making the total 3 for this design/board combo.

[Sanni Cart Reader - Top - Mod v3.zip](https://github.com/sanni/cartreader/files/6832326/Sanni.Cart.Reader.-.Top.-.Mod.v3.zip)
